src/histolab/scorer.py

```diff
@@ -18,5 +22,61 @@ def __call__(self, tile: Tile) -> float:
 
 
 class RandomScorer(Scorer):
+    """Implement a Scorer that returns a random float score between 0 and 1."""
+
     def __call__(self, tile: Tile) -> float:
+        """Return the random score associated with the tile.
+
+        Parameters
+        ----------
+        tile : Tile
+            The tile to calculate the score from.
+
+        Returns
+        -------
+        float
+            Random score ranging between 0 and 1.
+        """
         return np.random.random()
+
+
+class NucleiScorer(Scorer):
+    r"""Implement a Scorer that estimates the presence of nuclei in an H&E-stained tile.
+
+    An higher presence of nuclei is associated with an higher scorer, following this
+    formula:
+
+    .. math::
+
+        score = nuclei\_ratio \cdot tanh(tissue\_ratio)
+    """
+
+    def __call__(self, tile: Tile) -> float:
+        """Return the nuclei score associated with the tile.
+
+        Parameters
+        ----------
+        tile : Tile
+            The tile to calculate the score from.
+
+        Returns
+        -------
+        float
+            Nuclei score
+        """
+        tissue_ratio = tile.tissue_ratio
```

Comment:
I'd avoid this statement

just return: `nuclei_ratio * np.tanh(tile.tissue_ratio)`